identity-iota/src/tangle/client_builder.rs

```diff
@@ -36,6 +39,12 @@ impl ClientBuilder {
     self
   }
 
+  /// Disables message compression.
+  pub fn disable_compression(mut self) -> Self {
+    self.encoding = DIDMessageEncoding::Json;
+    self
+  }
```

Comment:
I still think this should be `set_compression` that takes the `DIDMessageEncoding` enum for forwards compatibility.

Reply:
I changed it now. 
The idea for using `disable_compression`  is that it implicitly means that the default is enabling compression + it is called on a builder and a config, so disabling compression then enabling it doesn't make much sense.

Reply:
Thanks for the change. 

I agree `disable_compression` is more explanative for examples but if/when more than two options are available in the future I would prefer to use the same `set_compression` function to choose between them. 
Come to think of it, the function should actually be called `set_encoding` or `set_message_encoding` since it sets the `DIDMessageEncoding`. Yeah I should have really picked up on the naming thing before, my bad.